IFC4x3/Properties/a/AssessmentFrequency_3Y5J0kLVfDHfMH8$QTwre7/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="AssessmentFrequency_3Y5J0kLVfDHfMH8QTwre7" Name="AssessmentFrequency" UniqueId="e215302e-55fa-4d46-9591-23f69deb5a07" PrimaryDataType="IfcTime" />
```

Comment:
i think it should be IfcTimeMeasure: 

- IfcTime is a particular point in time
- IfcTimeMeasure like 6 months or 1 year

Reply:
Yup IfcTime is a particular time in a day.

Reply:
```suggestion
<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="AssessmentFrequency_3Y5J0kLVfDHfMH8QTwre7" Name="AssessmentFrequency" UniqueId="e215302e-55fa-4d46-9591-23f69deb5a07" PrimaryDataType="IfcTimeMeasure" />
```
```suggestion
<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="AssessmentFrequency_3Y5J0kLVfDHfMH8QTwre7" Name="AssessmentFrequency" UniqueId="e215302e-55fa-4d46-9591-23f69deb5a07" PrimaryDataType="IfcTime" />
```

Reply:
```suggestion
<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="AssessmentFrequency_3Y5J0kLVfDHfMH8QTwre7" Name="AssessmentFrequency" UniqueId="e215302e-55fa-4d46-9591-23f69deb5a07" PrimaryDataType="IfcTimeMeasure" />
```